docker-compose.yml

```diff
@@ -289,10 +289,11 @@ services:
   nginx:
     image: nginx:latest
     volumes:
-      - ./../blue-stream-client/dist/blue-stream:/usr/share/nginx/html
       - ./nginx/nginx.conf:/etc/nginx/nginx.conf
     ports:
       - "80:80"
+    depends_on:
+      - angular-client
```

Comment:
This is not correct.
When you run with `ng serve` you are not depending on `angular-client` in fact you don't even need it up.